src/main/java/net/teumteum/meeting/service/MeetingService.java

```diff
@@ -25,14 +27,20 @@ public MeetingResponse getMeetingById(Long meetingId) {
     }
 
     @Transactional(readOnly = true)
-    public List<MeetingResponse> getMeetings(Long cursorId, Pageable size) {
-        var existMeetings = cursorId == 0 ?
-                meetingRepository.findByPromiseDateTimeGreaterThanOrderByIdDesc(size, LocalDateTime.now()) :
-                meetingRepository.findByIdLessThanEqualAndPromiseDateTimeGreaterThanOrderByIdDesc(cursorId, size, LocalDateTime.now());
-
-        return existMeetings.stream()
-                .map(MeetingResponse::of)
-                .toList();
+    public Slice<Meeting> getMeetingsBySpecification(Pageable pageable, Topic topic, String meetingAreaStreet,
+                                                     Long participantUserId, String searchWord, boolean isOpen) {
+
+        Specification<Meeting> spec = MeetingSpecification.withIsOpen(isOpen);
+
+        if (topic != null) spec = spec.and(MeetingSpecification.withTopic(topic));
+        else if (meetingAreaStreet != null) spec.and(MeetingSpecification.withAreaStreet(meetingAreaStreet));
+        else if (participantUserId != null)
+            spec = spec.and(MeetingSpecification.withParticipantUserId(participantUserId));
+        else if (searchWord != null)
+            spec = MeetingSpecification.withSearchWordInTitle(searchWord).or(MeetingSpecification.withSearchWordInIntroduction(searchWord))
+                    .and(MeetingSpecification.withIsOpen(isOpen));
+
```

Comment:
`반영하면 좋고 반영하지 않아도 괜찮음`

```suggestion
        if (topic != null) {
            spec = spec.and(MeetingSpecification.withTopic(topic));
        }
        else if (meetingAreaStreet != null) {
            spec.and(MeetingSpecification.withAreaStreet(meetingAreaStreet));
        }
        else if (participantUserId != null) {
            spec = spec.and(MeetingSpecification.withParticipantUserId(participantUserId));
        }
        else if (searchWord != null) {
            spec = MeetingSpecification.withSearchWordInTitle(searchWord).or(MeetingSpecification.withSearchWordInIntroduction(searchWord))
                    .and(MeetingSpecification.withIsOpen(isOpen));
        }

```